client/state/plugins/installed/selectors.js

```diff
@@ -47,11 +47,21 @@ export function isRequesting( state, siteId ) {
 	return state.plugins.installed.isRequesting[ siteId ];
 }
 
+export function isRequestingAll( state ) {
+	if ( typeof state.plugins.installed.isRequestingAll.all === 'undefined' ) {
```

Comment:
The way we perform this check demonstrates that we can simplify the `isRequestingAll` reducer a bit, as I commented above.